README.md

```diff
@@ -1,2 +1,65 @@
 # vue-money-amount
-Displays a stylable money amount according to the user's preferred formatting.
+Displays a stylable money amount and currency according to the user's preferred formatting.
+
+## Installation
+
+### Using yarn
+
+`yarn add vue-money-amount`
+
+### Using npm
+
+`npm i --save vue-money-amount`
+
+## Usage
+
+```js
+import VueMoneyAmount from 'vue-money-amout';
+
+Vue.component('vue-money-amout', VueMoneyAmount);
+```
+
+```html
+<money-amount :amount="amount" :currency="currency" :locale="locale" />
+```
+
+## Props API
+
+| Props                     | Type            | Required | Default             |
+|---------------------------|-----------------|----------|---------------------|
+| amount                    | Float           | true     | -                   |
+| currency                  | String          | true     | -                   |
+| locale                    | String          | no       | User locale         |
+| isFractionated            | Boolean         | no       | true                |
+| minimumFractionDigits     | Int             | no       | 0                   |
+
```

Comment:
Can you add a little note below the table about `isFractionated` and `minimumFractionDigits`? Because those might be the ones that can raise questions.

Reply:
Done, can you check if it suits you ?